contrib/database/sql/sql.go

```diff
@@ -250,3 +256,25 @@ func processOptions(cfg *config, driverName string, driver driver.Driver, dsn st
 	}
 	cfg.checkDBMPropagation(driverName, driver, dsn)
 }
+
+// leaving interval as a param in case we'd want it to be configurable
+func pollDBStats(interval time.Duration, db *sql.DB) {
+	for range time.NewTicker(interval).C {
```

Comment:
```suggestion
func pollDBStats(interval time.Duration, db *sql.DB) {
	// The interval will be a function parameter to allow for
	// potential configuration in the future.
	for range time.NewTicker(interval).C {
```
(just to keep it out of the godoc, since if it's there, I would expect it describes what the function does)